### ISSUE TITLE
Managing type collections

### ISSUE THREAD

**Issue by ekrimmel** [paleo-data/happy-hours#29]
## What is your idea for a happy hour topic?
Special considerations for managing data related to type collections?

## Who could be involved in leading this happy hour?
_[add text here - feel free to suggest yourself or others]_

## Are there any timing considerations?
_[add text here - e.g. needs to happen before xyz conference, after field season, ASAP, etc.]_

## Next steps
This issue captures a happy hour topic idea. Once you are ready to finalize an idea and put it on the PDWG happy hour schedule, please create a new issue using the [Plan a happy hour](https://github.com/paleo-data/happy-hours/issues/new?assignees=&labels=PDWG&projects=&template=plan-happy-hour.md&title=%5Breplace+this+text+with+the+happy+hour+topic%5D) template. **Then, close this issue.**

**Comment by ekrimmel** [comment]
Aligns with priorities for Alejandra and Suzanne in 2025
